### PR TITLE
change the color AND SO ON again

### PR DESCRIPTION
**Pull request by Jasonfintech**
change the color AND SO ON again

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,17 @@
   <head>
     <title>NBYG Do</title>
   </head>
-  <body>
-    <h1>YGDO</h1>
+<body bgcolor="00FFFF">
```
Comment (laosb) [inline index.html]: Not valid indent.

File: index.html [hunk 2]
```diff
@@ -2,9 +2,15 @@
   <head>
     <title>NBYG Do</title>
   </head>
-  <body>
+  <body bgcolor="00FFFF">
     <h1>YGDO</h1>
+    <hr>
+    <div style="color:#0000FF">
+      <h3>welcome</h3>
+      <p>This is a paragraph.</p>
```
Comment (laosb) [inline index.html]: Need content instead of placeholders.

File: index.html [hunk 3]
```diff
@@ -2,9 +2,15 @@
   <head>
     <title>NBYG Do</title>
   </head>
-  <body>
+  <body bgcolor="00FFFF">
```
Comment (laosb) [inline index.html]: `bgcolor` is deprecated. Use `style="background: #00ffff;"` instead.

File: index.html [hunk 4]
```diff
@@ -2,9 +2,17 @@
...
+  <body style="background: #00ffff;">
     <h1>YGDO</h1>
+    <hr>
+    <div style="color:#0000FF">
+      <h3>welcome</h3>
+      <p>Yinzhou Ningbo high school is invested by the district government over a billion yuan of new high-grade all boarding school public schools. 
+        <br>Is located in Yinzhou City, Ningbo city center lot. </br>
+      </p>
+    </div>
     <a href="http://www.nbyg.net">This is a link</a>
```
Comment (laosb) [inline index.html]: Description.

**Comment by Jasonfintech** [comment]
@laosb

**Comment by laosb** [comment]
LGTM.

**Pull request merged**